docs/integration.md

```diff
@@ -43,42 +43,39 @@
 8. Copy and modify configs from kubespray `group_vars` folder to corresponging `group_vars` folder in your existent project.
 You could rename *all.yml* config to something else, i.e. *kubespray.yml* and create corresponding group in your inventory file, which will include all hosts groups related to kubernetes setup.
 
-9. Modify your ansible inventory file by adding mapping of your existent groups (if any) to kubespray naming.  
+9. Modify your ansible inventory file by adding mapping of your existent groups (if any) to kubespray naming.
    For example:
    ```
      ...
      #Kargo groups:
      [kube-node:children]
      kubenode
-     
+
      [k8s-cluster:children]
      kubernetes
-     
+
      [etcd:children]
      kubemaster
      kubemaster-ha
-     
+
      [kube-master:children]
      kubemaster
      kubemaster-ha
-     
-     [vault:children]
-     kube-master
-     
+
      [kubespray:children]
      kubernetes
      ```
      * Last entry here needed to apply kubespray.yml config file, renamed from all.yml of kubespray project.
 
-10. Now you can include kargo tasks in you existent playbooks by including cluster.yml file: 
+10. Now you can include kargo tasks in you existent playbooks by including cluster.yml file:
```

Comment:
why there is kargo mentioned?

Reply:
I haven't changed it, just trailing a white space removed with my editor. 
Probably an old document :)